cli/azd/pkg/output/ux/list_as_text.go

```diff
@@ -17,14 +17,15 @@ import (
 func ListAsText(items []string) string {
 	count := len(items)
 	if count < 1 {
-		log.Panic("calling itemsCountAsText() with empty list.")
+		log.Panic("calling ListAsText() with empty list.")
 	}
 
 	if count == 1 {
 		return items[0]
 	}
 
 	if count == 2 {
+		//nolint:gosec // G602: slice index out of range - false positive, we know the slice has at least 2 elements
```

Comment:
could not think of a cleaner way to rewrite the function, `gosec` isn't correctly inferring `count` refers to `len(items)` in this usage.

Reply:
I wonder if a `switch` on `len(items)` would help the analysis here?  🤷.

Reply:
The `switch` didn't work -- looks like `gosec` isn't following the temp variable assignment. The one way I did find to fix this is to have `if len(items) ==2`. Supposedly, with compiler optimization, repeated `len(x)` calls should incur the same cost, but I'm just going to leave it as-is for now.